### PR TITLE
[studio] simplify onboarding

### PR DESCRIPTION
**Pull request by mohab-sameh**
Simplify onboarding text.

### CODE REVIEW

File: apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx [hunk 1]
```diff
@@ -4,22 +4,22 @@ const steps = [
   {
     element: "codemod",
     description:
-      "<p>This section is dedicated to codemods.</p><p>Codemods are tools used to automate code modifications and refactorings. They allow developers to make large-scale changes across a codebase quickly and consistently.</p><p> Here, you can see an example codemod script that transforms code from a 'Before' state to an 'After' state using jscodeshift.</p><p> You can modify the codemod's code here.</p>",
+      "<p>This section is dedicated to codemods.</p><p>Codemods are tools used to automate code modifications and refactorings. They allow developers to make large-scale changes across a codebase quickly and consistently.</p><p>The codemod editor allows you to see and modify the codemod you are building.</p>",
```
Comment (alexbit-codemod) [inline apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx:7]: too much text. folks who are here already know what codemod is.

what they need to know here is that this is the engine, which you are not talking about here.
make it super super concise... too many words here.

i can share my thoughts later but feel free to use your best judgement and ship. we iterate with user feedback.

File: apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx [hunk 2]
```diff
@@ -4,22 +4,22 @@ const steps = [
...
   },
   {
     element: "before-and-after-panels",
     description:
-      "<p>These panels display the code before and after applying a codemod. The 'Before' panel shows the original code, while the 'After' panel displays the expected output after transformation.</p><p> This visual comparison helps users understand the changes made by the codemod, ensuring the modifications meet the desired outcome.</p>",
+      "<p>The before/after panels allow you to specify how the code should look like before and after transformation.</p><p>To get started, insert your current code into the 'Before' panel, and how you want your code to look like after running the codemod into the 'After' panel.</p>",
```
Comment (alexbit-codemod) [inline apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx:12]: tooooo many words.

"This is your input, in the form of before/after test fixtures and comments describing the transformation logic."

File: apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx [hunk 3]
```diff
@@ -4,22 +4,22 @@ const steps = [
...
   },
   {
     element: "assistant",
     description:
-      "<p>The assistant section is available for logged users only and provides support and guidance throughout the codemod creation and execution process.</p> <p>It leverages AI to help users generate codemod scripts, troubleshoot issues, and refine their code transformations. This assistant is an invaluable resource for both novice and experienced developers looking to optimize their coding workflow. In the 'Debug' tab the codemods errors are listed and output of the console (<code>console.log</code> in codemod editor can be used)</p>",
+      "<p>The AI assistant provides support and guidance throughout the codemod creation and execution process by generating codemod scripts, troubleshooting issues, and incrementally refining code transformations.</p><p>The 'Debug' tab lists the codemod workflow and potential errors.</p>",
```
Comment (alexbit-codemod) [inline apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx:17]: too many not so much helpful words... instead:

"Assistant section includes AI, and for advanced users: AST and debugger. 
AI requires login. Start by clicking the "Generate codemod with AI" button."

File: apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx (hunk 1 above)
Comment (grzim) [reply apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx:7]: maybe in the intro we can also place some links to docs? Or even link the definition of codemod for folks who don't know the basics

File: apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx (hunk 2 above)
Comment (grzim) [reply apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx:12]: maybe as a last step, we can add FAQ or something like this for anyone who wants to dig in deeper?

File: apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx (hunk 1 above)
Comment (alexbit-codemod) [reply apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx:7]: Link to generic docs should already be in the studio. Engine-specific guides are nice but make this feature complicated. Let's keep it simple for now we iterate with user feedback later.

File: apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx (hunk 2 above)
Comment (alexbit-codemod) [reply apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx:12]: The product should be so intuitive that deep dive is not needed. Anything specific u have in mind? If not let's just keep it as short and simple as possible

File: apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx (hunk 1 above)
Comment (mohab-sameh) [reply apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx]: updated. Shortened and mentioned `jscodeshift` and `ts-morph` as the supported engines to use in studio. Please check and lmk if you like this more.

File: apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx [hunk 4]
```diff
@@ -29,12 +29,12 @@ const steps = [
...
     element: "output-panel",
     description:
-      "The output panel dynamically reflects changes made in the 'Before' or codemod editor. As users edit the code or the codemod script, the output panel updates in real-time to show the transformed code. This immediate feedback loop helps users quickly identify and correct any issues in their codemod logic.",
+      "<p>The output panel shows how the codemod in the editor affects the 'Before' code snippet in real-time.</p><p>A correct output should be identical to the 'After' code snippet, without introducing false positives or negatives. This immediate feedback loop helps users quickly identify and correct any issues in their codemod logic.</p>",
   },
   {
     name: "Feedback",
     description:
-      "Any feedback? Create an issue in our GitHub repo - <a style='color: blue' href='https://github.com/codemod-com/codemod/'>https://github.com/codemod-com/codemod/</a> ",
+      "Any feedback? <a style='color: blue' href='https://github.com/codemod-com/codemod/'>Create an issue in our GitHub repo</a>",
```
Comment (alexbit-codemod) [inline apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx:37]: Can come as a follow up, Save users one click by sending them to a create issue page

**Review by alexbit-codemod — APPROVED** [review]

**Pull request merged**